HW_1/task_8.py

```diff
@@ -0,0 +1,20 @@
+# Нарисовать в консоли ёлку спросив
+# у пользователя количество рядов.
+# Пример результата:
+# Сколько рядов у ёлки? 5
+#     *
+#    ***
+#   *****
+#  *******
+# *********
+
+def drawing_christmas_tree(size):
+    whitespace = size - 1
+    stars = "*"
+    for i in range(0, size):
```

Comment:
0 можно не передавать
```suggestion
    for i in range(size):
```